region/maridia/inner-pink/Colosseum.json

```diff
@@ -776,6 +805,35 @@
         }
       }
     },
+    {
+      "link": [3, 3],
```

Comment:
[2, 2] isn't hard with an angle down shot while walking off the platform.
I liked doing walk off shots for 1 and 3 too tbh.

Reply:
2 seems good, but the drop down method seems much worse to me. I added it as a note.